includes/admin/evf-admin-functions.php

```diff
@@ -384,12 +384,18 @@ function everest_forms_panel_field( $option, $panel, $field, $form_data, $label,
 			break;
 	}
 
+	if( ! empty( $args['smarttags'] ) ){
+		$smarttags_class = 'evf_smart_tag';
+	} else {
+		$smarttags_class = '';
+	}
```

Comment:
This condition can be check cleaverly if you do follow this step:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#clever-code

Reply:
Fixed :)